app/config/storage/inputs.php

```diff
@@ -4,7 +4,6 @@
     // Accepted inputs files
     "jpg" => "image/jpeg",
     "jpeg" => "image/jpeg",
-    "gif" => "image/gif",
```

Comment:
idea why this is removed? `imagick` issues?

Reply:
Yep I think it was causing memory issues